src/AppBundle/Controller/FrontController.php

```diff
@@ -10,6 +10,10 @@
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\Request;
 
+use AppBundle\RSS\Parser;
+use AppBundle\RSS\Template;
```

Comment:
``Template`` and ``DefaultTemplate`` are not called in this file, these ``use`` statements are thereby useless.